### PR TITLE
Android can now save and restore tox_save

### PR DESCRIPTION
**Pull request by GrayHatter**

### CODE REVIEW

**Comment by tsudoko** [comment]
tox_info?

**Comment by GrayHatter** [comment]
tox_save*

File: tox.c [hunk 1]
```diff
@@ -351,8 +351,12 @@ static _Bool load_save(Tox *tox)
                         /* Well, lets try the current directory... */
                         data = file_raw("tox_save", &size);
                         if(!data) {
-                            /* F***it I give up! */
-                            return 0;
+                            debug("trying android hack...");
+                            data = file_raw("/data/data/tox.utox/files/tox_save.tox", &size);
```
Comment (irungentoo) [inline tox.c]: why is this necessary?

Isn't that the path datapath() returns?

Comment (GrayHatter) [inline tox.c]: It's not anymore, I left that in because I was tired last night.

It was only in there while I was trying to get data_path to return
correctly.

You can remove it, or I can if you'd like?
